ADAL/src/ADAuthenticationResult+Internal.m

```diff
@@ -275,7 +276,10 @@ + (ADAuthenticationResult*)resultFromBrokerResponse:(MSIDBrokerResponse *)respon
     
     BOOL isMRRT = response.tokenResponse.isMultiResource;
     
-    MSIDConfiguration *config = [[MSIDConfiguration alloc] initWithAuthority:[NSURL URLWithString:response.authority] redirectUri:nil clientId:response.clientId target:response.resource];
+    __auto_type authorityFactory = [MSIDAuthorityFactory new];
+    __auto_type authority = [authorityFactory authorityFromUrl:[response.authority msidUrl] context:nil error:nil];
```

Comment:
nit: please don't use msidUrl as it will be removed by JK

Reply:
done